.github/config/integration-testing-scenarios/multitenancy.yaml

```diff
@@ -0,0 +1,3 @@
+matrix:
```

Comment:
We use another logic here where each version has its own CI config, and we can control what should run there.

Example:
https://github.com/camunda/camunda-platform-helm/blob/7282631e166248f38b00ebd6b05e8c7013ed1ad4/charts/camunda-platform-8.7/test/ci-test-config.yaml